girder_volview/web_client/views/open.js

```diff
@@ -16,10 +16,10 @@ export function addButton($el, parentSelector) {
     return button;
 }
 
-const volViewPath = `static/built/plugins/volview/index.html`;
+const volViewPath = '/volview';
```

Comment:
This changes the volview app to be served out of `/volview` rather than the old path.